vtest/accidental-12.mscx

```diff
@@ -104,13 +105,14 @@
         <defaultClef>F</defaultClef>
         </Staff>
       <trackName>Piano</trackName>
-      <Instrument>
+      <Instrument id="piano">
```

Comment:
This is correct. I just wondering why it isn't added before :open_mouth: since it should be here since the score ordering is merged.

Reply:
It's possible this had been saved in 3.6 before that merge.